### PR TITLE
Rewrite lottery page and fix CSRF

### LINKED ISSUE #240: Lottery signup/signoff is vulnerable to CSRF
### Feature or Bug?
- [ ] Feature
- [X] Bug

### Bug
It's not even a POST request. Only the very predictable and not-user-unique URL is needed.

### PR DESCRIPTION
**Pull request by HON95**
### Status

- [x] Done
- [ ] Work in progress
- [ ] Needs testing from others

### Changes
- Added list of participants and winners on lottery page.
- Added buttons to open and close raffles, including a permission to do so.
- Merged lottery info page and draw page.
- Removed link to lottery drawing from navbar.
- Fixed lottery CSRF vulnerability (fixes #240).
- Added optional payment enforcement to lotteries.

### Screenshots
[Screenshots](https://imgur.com/a/8aSCsrf)

### CODE REVIEW

File: files/css/custom.css [hunk 1]
```diff
@@ -452,3 +452,16 @@ div#btn-profile {
 .label, .label a {
     color: #FFF;
 }
+
+.lottery-buttons {
```
Comment (MathiasFl) [inline files/css/custom.css]: UInødvendig å ha samme css regel på to forskjellige selectors, slå sammen og bruk samme classenavn ex. .lottery-text-center.

File: locale/nb/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -1701,15 +1744,20 @@ msgstr "Må være unikt!"
 
 #, python-format
 msgid ""
-"%(inviter)s has invited you to join team <a href=\"%(team_url)s\">%(team)s</"
-"a>. <br /> You can accept or decline the invitation on the team page."
+"<strong>%(inviter)s</strong> has invited you to join team <a href="
```
Comment (MathiasFl) [inline locale/nb/LC_MESSAGES/django.po]: rel="noopener noreferrer"

File: locale/nb/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -1701,15 +1744,20 @@ msgstr "Må være unikt!"
...
+"\"%(team_url)s\"><strong>%(team)s</strong></a>. <br /> You can accept or "
+"decline the invitation on the team page."
 msgstr ""
-"%(inviter)s har invitert deg til å bli med på lag <a href=\"%(team_url)s\">"
-"%(team)s</a>. <br /> Du kan akseptere eller avslå invitasjonen på lagsiden."
+"<strong>%(inviter)s</strong> har invitert deg til å bli med i lag <a href="
```
Comment (MathiasFl) [inline locale/nb/LC_MESSAGES/django.po]: rel="noopener noreferrer"

File: files/css/custom.css (hunk 1 above)
Comment (HON95) [reply files/css/custom.css]: Det blir jo veldig å blande struktur og stil, men okay.

Comment (HON95) [reply files/css/custom.css]: Done.

**Pull request merged**